scripts/generate_js_definitions.sh

```diff
@@ -55,3 +55,12 @@ npm run build
 # Generate ethereum-utils packed tgz
 cd dist
 npm pack
+
+# build recovery-sdk
+cd "../../recovery-sdk"
+npm install
+npm run build
+
+# Generate recovery-sdk packed tgz
+cd dist
+npm pack
```

Comment:
Looks like this got missed when the recovery-sdk was added; not needed in CI because it uses previously built artifacts, but necessary if you want to run E2E tests locally.